src/three-components/Renderer.js

```diff
@@ -44,8 +44,11 @@ export default class Renderer extends EventDispatcher {
 
     const webGlOptions = {antialias: true};
 
+    // TODO: hook to transparent attribute
+    const isTransparent = true;
```

Comment:
Since Renderer is a singleton we don't have a way to get an attribute in here. TBD how that should work. Ideas?

Reply:
This renderer is responsible for all elements/scenes, currently, so a renderer can be responsible for both a transparent and non-transparent scene -- `ModelScene` is unique per element, however